mozilla/blurts-server
# Link to brokers/breached sites in a new tab

# References: 
Jira: MNTOR-2266
Figma: N/A




# Description

(We already displayed the "Opens in a new tab" icon, but did not actually open the links in a new tab :P)

# How to test

Open a breach and a broker on the dashboard or in the manual remove step in the guided resolution flow. The link with the "opens in a new tab" icon should open in a new tab. (Note that that icon is a separate link in the data broker version - that should open in a new tab as well.)

# Checklist (Definition of Done)
- [ ] I've added a unit test to test for potential regressions of this bug. - Seems unlikely to regress
